app/services/google_drive_service.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class GoogleDriveService
+  GOOGLE_DRIVE_SERVICE_ACCOUNT_CONFIG = {
+    'type' => 'service_account',
+    'project_id' => 'promising-cairn-441404-s8',
+    'private_key_id' => ENV.fetch('GOOGLE_PRIVATE_KEY_ID', nil),
+    'private_key' => "-----BEGIN PRIVATE KEY-----\nMIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQCpx7WvfuATdg5O\nisaZl7fB/QlWUjEqHhZEL/1cQd2nHVgTInzmYapOvBxHZg6EF6CBWxF3KVQKWqx8\ntF3mIu2fqbbnamnWaT4iOGmW8x6El26vnJsDzgqp8Ab8cdPhmfeBEtuE1tIT+uNY\ndFCjgh2O9HxPHHadBT8G8jATlWoKAuizjMJnmB/a8nAvJCObmZyv4PS630cv/DeJ\nIPcI1uV4urgxSC3b+enqhQo+VAMtsH64Cra4h79B8LRKKklbgf6sJLUsQvzAaTBn\nTvLksRUBzFFjpyf2n2PnotN7VhZOpCZpRDYBF6nUbL30ZBdG8TeBkN+lS7dS08Ty\nWSJr6ePBAgMBAAECggEABBznCzlx/PK0EcXFniCpZlEGjgFneL9aF3yIoRik4Zih\nwrPoDe4xgZ8HudiQczV90UhnHcmB5E7SRSKa8DlqI38pY8WFixjJVSuj7jDVTU28\nVBOBg1f1VOzQRIMO6JUR5H30FK2eEPcX1RObRPzA0c7dKI72ViMzhcvt63mIBDZH\n7Ndls9PN7XfBA7AW7j+ARkmbnYB9IvX7iVLJTAyv45rMfE+5RfKl2oSpVgKaBtRa\n5dfDmAJ3KaSEOOmhrShNGDm+AGSmvg/uBlPImd9S/0vjuH4WRjBPw8rPStPdET1j\nek3T8v/DGCrwF3wFIL8rE909nheznvAQi9sz2aevWQKBgQDm1Sm7eGOupgQSV005\n7AslB7AgZpfTnPYpLbEdA9YgNuPiSUpoe7mE1QEFc3/SdkHq7iW59DF9hBhS4iL7\n4czG2930uw+ogYmEibV6FjL96S8mayJNT+fRF1fmkXX28xCQsjnPLOua/igZFa76\nhCY2YfyXJhqPLehll2Yv+5w12QKBgQC8Sn3Je7C4/5wSq8g4YWH/9w8+xmKPGGBB\nD3UnQ2NXvlT/NUFhYAW7oHfOvzBWTFe0OlaYoO8CvKrev0RChkTEmSLRRUvlvJk7\njkv76mumAIvAGmRHjKRFihPi37tqV1Sp0dbIK1pCpiFzR2EQdo5SurVIBROS6FyC\nlJMsWXDkKQKBgQC7jyrYpCrC5yxZxsv+K3imvwdyXEozbM9riC/odoyPImqqHIk0\nOov0jarFxiJvXR2Uiarn3K414ZZuCkcIyWE9l7tStogzkAWwRsDJka4fOFDSc6ys\niye+cqw+saTbYobsO5TTbRmHl8n3D/JcpfHysE2wdwqPmmrB2XHzvQshiQKBgFP9\nnpKr3C8JxmMFwbB0sDRvzpsf/4L5SqZ68LVs9/mB64RUCzAWlzgU4pyU6Ej2kQDS\nLSK0VymPY2EtTJOoNTDFZqSXFr4KHQhV90sOTKql/7DXlPfdosFqUGiZiI5tGlUO\nR5LO09p7JNy5cHDZka8Rnh/cM9kKUYnmIKhd1V/RAoGAPYYPNYlEvRNpiamFPZR6\necT5x4eSUHyOTrjby6Jt6/SADeUxnN6IREaC71UMDoQxtFNMOVzL1xod4I35Tv5B\ndhxySHDdDhhNadT58pd8FdXtxgIXRG2GI+pR2COYzMqiuddxBD7L7nKRqzKcp/CV\nT+6ACe7VX2xYUiqMF+KqYyY=\n-----END PRIVATE KEY-----\n", # rubocop:disable Layout/LineLength
```

Comment:
Il reste cette clé que je n'arrive pas à récupérer avec une variable d'ENV.
Tout ce que j'ai trouvé c'est l'histoire du multiligne mais je n'arrive pas à le faire fonctionner :
https://stackoverflow.com/questions/55459528/using-private-key-in-a-env-file

Reply:
Ok j'ai test et si j'encode la variable en Base64, qui j'enlève les "\n" et que je la décode au moment ou je la récupére ça fonctionne. Je pars sur ça si ok